packages/account/src/Helpers/utils.tsx

```diff
@@ -6,9 +6,10 @@ import {
     validName,
     IDV_NOT_APPLICABLE_OPTION,
 } from '@deriv/shared';
-import { localize } from '@deriv/translations';
+import { Localize, localize } from '@deriv/translations';
 import { ResidenceList, GetSettings, GetAccountStatus } from '@deriv/api-types';
 import { FormikErrors, FormikValues } from 'formik';
+import React from 'react';
```

Comment:
Please sort the import statements as per this doc https://github.com/binary-com/deriv-app/blob/master/docs/JavaScript/README.md#import-rules